galaxy/templates/configs-galaxy.yaml

```diff
@@ -16,7 +16,36 @@ data:
 {{- end }}
   {{- range $key, $entry := .Values.configs -}}
   {{- if $entry -}}
-  {{- $key | nindent 2 }}: |
-    {{- tpl (tpl $entry $) $ | nindent 4 -}}
-  {{- end }}
-  {{- end }}
+  {{- $key | nindent 4 }}: |
```

Comment:
@almahmoud Now that the libyaml change has been made: https://github.com/CloudVE/galaxy-docker-k8s/pull/11, it seems like we should convert this to the simplest version that works. Wasn't there some version you wrote that handled both but was fairly simple?

Reply:
Done!

Reply:
I also rebased everything into a single commit